backend/src/main/resources/db/migration/V12__created_at_audit_columns.sql

```
@@ -0,0 +1,30 @@
+-- Neither table has recorded its insert time since V1, and every row written without the column
+-- loses that fact for good — so it lands now, ahead of the first feature that reads it (#226).
+--
+-- Backfill: the earliest price record reachable from the row is the best estimate — a listing existed
+-- at least as early as its first observation, a product at least as early as its first listing. Rows
+-- with no history get now(). Add nullable, backfill, then constrain, so the dev DB migrates too.
+
+ALTER TABLE tracked_item ADD COLUMN created_at timestamp with time zone;
+
+UPDATE tracked_item ti
+SET created_at = COALESCE(
+    (SELECT MIN(pr."timestamp") FROM price_record pr WHERE pr.tracked_item_id = ti.id),
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)